app/src/main/res/layout/topic_preview_fragment.xml

```diff
@@ -0,0 +1,288 @@
+            android:layout_marginStart="12dp"
+            android:fontFamily="sans-serif"
+            android:gravity="center_vertical"
+            android:text="@string/topic_preview_download_topic"
+            android:textColor="@color/oppiaPrimaryText"
+            android:textDirection="locale"
+            android:textSize="18sp"
+            app:layout_constraintBottom_toBottomOf="@+id/topic_preview_download_image_view"
+            app:layout_constraintStart_toEndOf="@+id/topic_preview_download_image_view"
+            app:layout_constraintTop_toTopOf="@+id/topic_preview_download_image_view" />
+
+          <TextView
+            android:id="@+id/download_size_text_view"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginStart="8dp"
+            android:layout_marginEnd="32dp"
+            android:fontFamily="sans-serif"
+            android:text="@{String.format(@string/topic_download_text, viewModel.topicSize)}"
+            android:textColor="@color/oppiaPrimaryText"
+            android:textDirection="locale"
+            android:textSize="18sp"
+            android:textStyle="italic"
+            app:layout_constraintBottom_toBottomOf="@+id/topic_preview_download_image_view"
+            app:layout_constraintStart_toEndOf="@+id/download_story_count_text_view"
+            app:layout_constraintTop_toTopOf="@+id/topic_preview_download_image_view" />
+
+          <TextView
+            android:id="@+id/stories_heading"
+            style="@style/Heading2"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="36dp"
+            android:text="@{@plurals/topic_preview_story_heading(viewModel.topic.storyCount, viewModel.topic.storyCount)}"
+            app:layout_constraintStart_toStartOf="@+id/topic_preview_download_image_view"
+            app:layout_constraintTop_toBottomOf="@+id/download_story_count_text_view"
+            tools:text="Stories you can play" />
+
+          <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/story_summary_recycler_view"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginTop="8dp"
+            android:layout_marginEnd="32dp"
+            android:clipToPadding="false"
+            android:overScrollMode="never"
+            android:paddingBottom="16dp"
+            android:scrollbars="none"
+            app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="@+id/topic_preview_download_image_view"
+            app:layout_constraintTop_toBottomOf="@+id/stories_heading"
+            app:list="@{viewModel.storyItemList}" />
+
+          <TextView
+            android:id="@+id/skills_heading"
```

Comment:
Aren't these actually subtopics, not skills? I think we just refer to them as 'skills' in the UI to simplify things for users, but technically they're subtopics.